### ISSUE TITLE
Slider a11y bug - must have an accessible name

### ISSUE THREAD

**Issue by adrianaferrugento** [radix-ui/primitives#2127]
### Current Behavior

Getting an accessibility error with the slider. It doesn't have an accessible name. I believe the aria-label is being set on the wrong element.

![Screenshot 2023-05-03 at 17 53 00](https://user-images.githubusercontent.com/76945696/235986771-e8f1d05f-6bff-4517-9152-ee5a6f6dfac5.png)

### Expected behavior

The slider would have the aria-label set on the correct element as it's implemented in your code - https://github.com/radix-ui/primitives/blob/main/packages/react/slider/src/Slider.tsx#L599

### Reproducible example

You can see this in your example page. The element with `role="slider"` doesn't have an aria-label as expected. If you run axe on your page you'll see the same error.

https://www.radix-ui.com/docs/primitives/components/slider

<img width="1094" alt="Screenshot 2023-05-03 at 18 06 50" src="https://user-images.githubusercontent.com/76945696/235989158-edaa1a2b-fb21-458a-bc2e-9e3dee89deca.png">
<img width="1110" alt="Screenshot 2023-05-03 at 18 06 30" src="https://user-images.githubusercontent.com/76945696/235989160-2ace14f1-18e3-4c1c-bbec-b0bbe16450e8.png">

### Suggested solution

According to the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role)
> An accessible name is required. If the range's role is applied to an HTML [<input>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) element (or <meter> or <progress> element), the accessible name can come from the associated [<label>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label). Otherwise use [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) if a visible label is present or [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) if a visible label is not present.

My proposal is that the aria-label is applied to the element with `role="slider"`. I believe it's being overridden by this line - https://github.com/radix-ui/primitives/blob/main/packages/react/slider/src/Slider.tsx#L607.

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Radix Package(s) |    slider     |     1.1.1    |
| React            | n/a     |     latest    |
| Browser          |         |   Chrome      |
| Assistive tech   |         |         |
| Node             | n/a     |         |
| npm/yarn         |         |         |
| Operating System |         |         |

**Comment by benoitgrelard** [comment]
@adrianaferrugento,

This all works as expected, if you label yourself by passing `aria-label` to `Slider.Thumb`, your label does indeed end up on the `role="slider"` element.

However you are correct that the aria-label in our example demo on the site is passed to the wrong element, I will fix this separately now. Thanks for pointing that out!

**Comment by benoitgrelard** [comment]
Docs fix: https://github.com/radix-ui/website/pull/637
